### DISCUSSION TITLE
hoursInterval and minutesInterval options in config

### DISCUSSION THREAD

**Discussion by juangvega** [dmuy/MDTimePicker#34] [Ideas]
Hi @dmuy 

I've been using this plugin for several years now. It is amazing! Thank you for this!

Two quick features that I added are the option to set the interval for the hours and minutes to select. Let's say I only want the users to select times within 5-minute intervals. Ex. 1:00 PM, 1:05 PM... 

This is pretty straightforward to add: set the value in the config, being 1 the default. And change the for-loop that creates the hours/minutes elements to increment by this value.

This feature has been very useful for me; I guess it could be for others as well.

Thanks again for this work!

**Comment by dmuy** [comment]
Thank you @juangvega, I'm glad this project helped you out.

I'll take this feature into consideration when I make updates and such in the future. Unfortunately I can't focus on this right now, but if can request a PR I can take a look at it, that would be much help 👍
